### ISSUE TITLE
Couldn't achieved the performance described in the paper

### ISSUE THREAD

**Issue by yuanbin1224** [sitzikbs/3DmFV-Net#1]
I have trained the 3DmFV model several times using the default parameters, but i haven't achieved the performance described in the paper. My best classification accuracy on modelnet10 was about 93.3% and 88.9% on modelnet40. So, could you list your training config? Thank you.

**Comment by sitzikbs** [comment]
I am closing the issue since it is not a code issue. If you have further questions PM me. 

Regarding your question: the configuration is described in detail the paper and attached below. 

- Make sure you are using 8x8x8 Gaussians with 0.0156 sigmas (this is different than the defaults in the code here). 

- You should apply the data augmentation (on trainingset) : augment_scale, augment_translation, augment_jitter. (The other two are for robust training and is a bit of a tradeoff to the accuracy ) . 

- Use adam optimizer, 2048 points, a batch of 64 or higher, learning rate of 0.001, 0.7 decay rate

- Use 3DmFV with all 3 symmetric functions ( min, max and sum).
